examples/with-ably/package.json

```
@@ -0,0 +1,24 @@
+{
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "lint": "next lint"
+  },
+  "dependencies": {
+    "@ably-labs/react-hooks": "^2.0.1",
+    "ably": "^1.2.22",
+    "got": "^12.1.0",
+    "next": "latest",
+    "react": "18.1.0",
+    "react-dom": "18.1.0"
+  },
+  "devDependencies": {
+    "@types/react": "^18.0.12",
+    "eslint": "8.16.0",
+    "eslint-config-next": "latest",
+    "next-transpile-modules": "^9.0.0",
+    "typescript": "^4.7.3"
```

```suggestion
    "start": "next start"
  },
  "dependencies": {
    "@ably-labs/react-hooks": "^2.0.1",
    "ably": "^1.2.22",
    "got": "^12.1.0",
    "next": "latest",
    "react": "18.1.0",
    "react-dom": "18.1.0"
  },
  "devDependencies": {
    "@types/react": "^18.0.12",
    "next-transpile-modules": "^9.0.0",
    "typescript": "^4.7.3"
```

`@types/node` is also missing.